package.json

```diff
@@ -2,33 +2,32 @@
   "name": "@veupathdb/eda",
   "version": "1.8.3",
   "dependencies": {
-    "@emotion/react": "^11.4.1",
-    "@emotion/styled": "^11.3.0",
-    "@material-ui/core": "^4.11.3",
-    "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.58",
-    "@types/debounce-promise": "^3.1.3",
-    "@veupathdb/components": "^0.11.27",
-    "@veupathdb/coreui": "^0.13.0",
-    "@veupathdb/http-utils": "^1.1.0",
     "debounce-promise": "^3.1.2",
     "fp-ts": "^2.9.3",
     "io-ts": "^2.2.13",
     "lodash": "^4.17.21",
     "monocle-ts": "^2.3.11",
-    "react": "^17.0.1",
     "react-cool-dimensions": "^2.0.7",
-    "react-dom": "^17.0.1",
     "react-draggable": "^4.4.3",
     "react-resizable": "^3.0.4",
-    "react-router": "^5.2.1",
-    "react-router-dom": "^5.3.0",
     "react-table": "^7.7.0",
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
+    "@material-ui/core": "^4.11.3",
+    "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.58",
+    "@veupathdb/components": "^0.11.27",
+    "@veupathdb/coreui": "^0.13.0",
+    "@veupathdb/http-utils": "^1.1.0",
     "@veupathdb/study-data-access": "^0.2.0",
-    "@veupathdb/wdk-client": ">=0.5.11"
+    "@veupathdb/wdk-client": ">=0.5.11",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
```

Comment:
Perhaps we should allow React 16 as well?

Reply:
Yes, that's a good idea. I will work on making the version ranges for peerDependencies more forgiving soon.